packages/aws-arch/README.md

```diff
@@ -0,0 +1,90 @@
+# aws-arch (AWS Architecture)
+
+Library to provide metadata for AWS Services and AWS Resources.
+
+This package generates mappings between [@aws-cdk/cfnspec](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cfnspec) and [AWS Architecture Icons](https://aws.amazon.com/architecture/icons/), and resolves resource metadata between these systems to infer a normalized definition of an `AwsService` and `AwsResource`.
+
+The primary aim of this package is to provide a consistent mechanism for other libraries to retrieve naming and assets associated with CloudFormation resources, and by extension CDK resources.
+
+## Get Started
```

Comment:
Do we have the build/generate process documented anywhere i.e: run `npx projen generate`?

Reply:
I will add this to the README.md under a "Development" heading